packages/kcms/src/team/adaptor/validator/match.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from '@hono/zod-openapi';
+
+const finishStatus = ['goal', 'retired'] as const;
```

Comment:
```suggestion
const finishState = ['goal', 'finished'] as const;
```
だと思います。また、使いまわさないならL15に直接書いてしまってもいいでしょう。

Reply:
直接書きます。
notionでは`goal`と`retired`となっていますが上のように`goal`と`finished`とした方が良いですか？

Reply:
@laminne